feature/sessions/src/androidMain/kotlin/io/github/droidkaigi/confsched/sessions/components/ContextMenuProvider.android.kt

```diff
@@ -0,0 +1,15 @@
+package io.github.droidkaigi.confsched.sessions.components
+
+import androidx.compose.runtime.Composable
+import io.github.droidkaigi.confsched.sessions.grid.TimetableGridScope
+
+@Composable
+actual fun TimetableGridScope.ContextMenuProviderForDesktop(
+    isBookmarked: Boolean,
```

Comment:
Ultimately, since we cannot change the text without receiving the bookmark status from an external source, we are adding this argument.
If we use neutral text, it becomes impossible to tell whether the right-clicked item is bookmarked or not, which is extremely inconvenient.
Therefore, although it differs from the request, this argument is necessary in addition to adding the lambda.